data/jobs.txt

```diff
@@ -1483,6 +1483,169 @@ mission "Core Pirate Attack"
 		payment 200000
 		dialog "The government of <planet> pays you <payment> for helping to drive off the pirates."
 
+mission "Renegade Drones 1"
+	name "Neutralize broken Drone"
+	repeat
+	description "Destroy or disable the malfunctioning Navy drone."
+	minor
+	to offer
+		not "Renegade Drones 1: active"
+		not "Renegade Drones 2: active"
+		not "Renegade Drones 3: active"
+		random < 20
+		"combat rating" > 200
+	clearance
+	source "Pilot"
+	destination "Pilot"
+	on offer
+		conversation
+			`Walking past a group of Navy soldiers, you hear one shout out: "Not again, another drone seems to have a malfunction and attacks ships in orbit. Who's turn is it to shoot it down?"`
+			`Nobody answers.`
+			`What do you want to do?`
+			choice
+				`	(Raise your hand and offer to shoot it down.)`
+					goto yes
+				`	(Ignore them and walk on.)`
+					decline
+			label yes
+			`You raise your hand and one of the soldiers notices you.`
+			`"You? Why not. Try not to hurt yourself."`
+			`You got the job.`
+				accept
+	npc evade
+		government "Renegade"
+		personality heroic target vindictive nemesis marked
+			confusion 30
+		fleet
+			names "republic fighter"
+			variant
+				"Combat Drone" 1
+	on complete
+		payment 20000
+		dialog `A Navy soldier approaches you. "Well done!", and with a firm hand shake she hands you <payment>.`
+
+mission "Renegade Drones 2"
+	name "Neutralize broken Drones"
+	repeat
+	description "Destroy or disable the malfunctioning Navy drones."
+	minor
+	to offer
+		not "Renegade Drones 1: active"
+		not "Renegade Drones 2: active"
+		not "Renegade Drones 3: active"
+		random < 10
+		"Renegade Drones 1: done" > 3
+		"combat rating" > 600
+	clearance
+	source "Pilot"
+	destination "Pilot"
+	on offer
+		conversation
+			`Walking through the space port, you are approached by a Navy officer. "Captain <last>?" she asks. You nod. "I've heard you're quite good at shooting things from the sky. A group of drones we've tested experienced a malfunction. Now they rampage through the system attacking everything getting in their way. Unfortunately, we lack personnel at the moment. Would you be willing to help us out and shoot down the malfunctioning drones?"`
+			choice
+				`	"Sure, I always welcome targeting practice."`
+					accept
+				`	"No, sorry, I'm busy right now."`
+					decline
+	npc evade
+		government "Renegade"
+		personality heroic target vindictive nemesis
```

Comment:
I think both fleets are `target`, aren't they? I probably should use the same order of attributes for all fleets. Is there a canonical form? I've checked the other jobs, but I don't see a pattern.